Application/EdFi.Ods.Extensions.TPDM/Artifacts/PgSql/Data/Security/1010-TPDM-ResourceClaims.sql

```diff
@@ -1,487 +1,5254 @@
-where ResourceName in ('anonymizedStudent'
-, 'anonymizedStudentAssessments'
-, 'performanceMeasure'
-, 'performanceMeasureCourseAssociation'
-, 'professionalDevelopmentEvent'
-, 'recruitmentEvent'
-, 'staffTeacherPreparationProviderAssociation'
-, 'anonymizedStudentAssessment'
-, 'talentManagementGoal'
-, 'teacherCandidate'
-, 'teacherCandidateStudentGrowthMeasure'
-, 'teacherCandidateTeacherPreparationProviderAssociation'
-, 'anonymizedStudentSectionAssociation'
-, 'applicant'
-, 'applicantProspectAssociation'
-, 'anonymizedStudentEducationOrganizationAssociation'
-, 'anonymizedStudentCourseTranscript'
-, 'anonymizedStudentCourseAssociation'
-, 'anonymizedStudentAssessmentSectionAssociation'
-, 'anonymizedStudentAssessmentCourseAssociation'
-, 'anonymizedStudentAcademicRecord'
-, 'prospect'
-, 'performanceMeasureFact'
-, 'educationOrganizationCourseTranscriptFacts'
-, 'courseStudentFacts'
-, 'courseStudentAssessmentFacts'
-, 'courseStudentAcademicRecordFacts'
-, 'courseCourseTranscriptFacts'
-, 'staffEvaluationComponent'
-, 'staffEvaluationComponentRating'
-, 'staffApplicantAssociation'
-, 'openStaffPositionEvent'
-, 'employmentSeparationEvent'
-, 'employmentEvent'
-, 'educationOrganizationStudentAssessmentFacts'
-, 'educationOrganizationStudentAcademicRecordFacts'
-, 'completerAsStaffAssociation'
-, 'applicationEvent'
-, 'teacherPreparationProviderProgram'
-, 'teacherCandidateTeacherPreparationProviderProgramAssociation'
-, 'teacherCandidateStudentGrowthMeasureSectionAssociation'
-, 'teacherCandidateStudentGrowthMeasureEducationOrganizationAssociation'
-, 'teacherCandidateStaffAssociation'
-, 'teacherCandidateCourseTranscript'
-, 'teacherCandidateAcademicRecord'
-, 'staffTeacherPreparationProviderProgramAssociation'
-, 'staffStudentGrowthMeasureSectionAssociation'
-, 'staffStudentGrowthMeasureEducationOrganizationAssociation'
-, 'staffStudentGrowthMeasureCourseAssociation'
-, 'staffStudentGrowthMeasure'
-, 'staffProspectAssociation'
-, 'fieldworkExperienceSectionAssociation'
-, 'staffEvaluationRating'
-, 'staffEvaluationElementRating'
-, 'staffEvaluationElement'
-, 'staffEvaluation'
-, 'sectionStudentFacts'
-, 'sectionStudentAssessmentFacts'
-, 'sectionStudentAcademicRecordFacts'
-, 'sectionCourseTranscriptFacts'
-, 'rubricLevelResponseFacts'
-, 'rubricLevelResponse'
-, 'rubricLevel'
-, 'rubric'
-, 'performanceMeasureFacts'
-, 'teacherCandidateStudentGrowthMeasureCourseAssociation'
-, 'application'
-, 'professionalDevelopmentEventAttendance');
-
-end $$
+DO $$
+DECLARE
+    _applicationId INTEGER;
+    _claimId INTEGER;
+    _claimName VARCHAR(2048);
+    _parentResourceClaimId INTEGER;
+    _existingParentResourceClaimId INTEGER;
+    _claimSetId INTEGER;
+    _claimSetName VARCHAR(255);
+    _authorizationStrategyId INTEGER;
+    -- Well-known values
+    _createActionId INTEGER := 1;
+    _readActionId INTEGER := 2;
+    _updateActionId INTEGER := 3;
+    _deleteActionId INTEGER := 4;
+    _claimIdStack INTEGER ARRAY;
```

Comment:
i know we talked about the naming convention, it should be snake case... eg application_id

Reply:
Done.